Yank/yank.py

```diff
@@ -316,12 +316,38 @@ def _create_phase(self, phase, reference_system, positions, atom_indices, thermo
         # Make a deep copy of the reference system so we don't accidentally modify it.
         reference_system = copy.deepcopy(reference_system)
 
+        # Store a serialized copy of the reference system.
+        from simtk.openmm import XmlSerializer
+        metadata['reference_sysem'] = XmlSerializer.serialize(reference_system)
```

Comment:
Is there a reason to store the systems both in the `metadata` and in the `thermodynamic_state` groups in the `.nc` file?